scraper/free_scraper/pipelines.py

```diff
@@ -46,12 +46,19 @@ def process_item(self, item, spider):
         """
         Process incoming items
         """
+        print "New Email Item: "
+        print item
+        print type(item)
+        print item.get("seats_data").get("general_remaining")
+        general_seats = item.get("seats_data").get("general_remaining")
+        # print "Num of seats: " + item["seats_data"].get("general_remaining", None)
+
         # Change the seat types accordingly
-        open_seats = bool(0 < int(item['seats_data']['general_seats']))
+        open_seats = bool(0 < general_seats)
         if open_seats:
```

Comment:
Can probably put this back into a single line:
```py
if general_seats > 0:
    # do things
```
Also be careful with this because I believe that `general_seats` [is a string](https://github.com/leesw98/FreeEyeOut/blob/413b7143b0d01f6fe06556e8146d57e885da6bae/scraper/free_scraper/spiders/parsers/course_parser.py#L99) and that could [cause some problems](https://stackoverflow.com/a/11767888) - might want to update the parser, but I could be wrong